docs(sources): add integrations directory with 31 source pages (ETL-980) (#739)
glassflow/clickhouse-etl: docs/app/sources/mysql/page.mdx

---
title: 'MySQL'
description: 'Stream changes from MySQL into ClickHouse with GlassFlow'
---
import { Callout } from 'nextra/components'

# MySQL

MySQL is a widely used open-source relational database optimized for web applications and transactional workloads. It is known for its simplicity, performance, and broad ecosystem support.

## Using MySQL with GlassFlow

GlassFlow can ingest changes from MySQL into ClickHouse with the same pipeline capabilities available to all GlassFlow sources — deduplication, joins, filtering, and stateless transformations.

<Callout type="info">

> **Available on request.** MySQL connectors are supported natively via the GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.

Reviewer: ```suggestion
**Available on request.** MySQL connectors are natively supported in GlassFlow Enterprise Edition. To get started, [join our GlassFlow Slack community](https://glassflowhub.slack.com/join/shared_invite/zt-349m7lenp-IFeKSGfQwpJfIiQ7oyFFKg) to chat directly with our team, or [reach out](https://www.glassflow.dev/integrations#contact) to us with your use case, expected volumes, and timeline.
```
